patches/remove-useless-ui.patch

```diff
@@ -0,0 +1,270 @@
+diff --git a/chrome/browser/resources/settings/people_page/people_page.ts b/chrome/browser/resources/settings/people_page/people_page.ts
+index 3ec954ed874f3..36e08094951bc 100644
+--- a/chrome/browser/resources/settings/people_page/people_page.ts
++++ b/chrome/browser/resources/settings/people_page/people_page.ts
+@@ -354,12 +354,7 @@ export class SettingsPeoplePageElement extends SettingsPeoplePageElementBase {
+ 
+   // <if expr="not chromeos_ash">
+   private computeShouldShowGoogleAccount_(): boolean {
+-    if (this.storedAccounts === undefined || this.syncStatus === undefined) {
+-      return false;
+-    }
+-
+-    return (this.storedAccounts!.length > 0 || this.isSyncing_()) &&
+-        !this.syncStatus!.hasError;
++    return false;
+   }
+   // </if>
+ 
+@@ -412,15 +407,7 @@ export class SettingsPeoplePageElement extends SettingsPeoplePageElementBase {
+   }
+ 
+   private shouldShowSyncAccountControl_(): boolean {
+-    // <if expr="chromeos_ash">
+     return false;
+-    // </if>
+-    // <if expr="not chromeos_ash">
+-    if (this.syncStatus === undefined) {
+-      return false;
+-    }
+-    return !!this.syncStatus!.syncSystemEnabled && this.signinAllowed_;
+-    // </if>
+   }
+ 
+   /**
+diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+index 00318322c8147..409f855a3a2b1 100644
+--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
++++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+@@ -417,9 +417,7 @@ export class SettingsPrivacyPageElement extends SettingsPrivacyPageElementBase {
+   override currentRouteChanged() {
+     this.showClearBrowsingDataDialog_ =
+         Router.getInstance().getCurrentRoute() === routes.CLEAR_BROWSER_DATA;
+-    this.showPrivacyGuideDialog_ =
+-        Router.getInstance().getCurrentRoute() === routes.PRIVACY_GUIDE &&
+-        this.isPrivacyGuideAvailable;
++    this.showPrivacyGuideDialog_ = false;
+ 
+     // Only record the metrics when the user navigates to the notification
+     // settings page that shows the entry point.
+@@ -608,8 +606,7 @@ export class SettingsPrivacyPageElement extends SettingsPrivacyPageElementBase {
+   }
+ 
+   private shouldShowAdPrivacy_(): boolean {
+-    return !this.isPrivacySandboxRestricted_ ||
+-        this.isPrivacySandboxRestrictedNoticeEnabled_;
++    return false;
+   }
+ 
+   // Only show Manage Topics page when PTB is enabled. If user is part of Mode B
+diff --git a/chrome/browser/resources/settings/privacy_page/security_page.html b/chrome/browser/resources/settings/privacy_page/security_page.html
+index 9a8fe2fe5df1a..e1b9698582af2 100644
+--- a/chrome/browser/resources/settings/privacy_page/security_page.html
++++ b/chrome/browser/resources/settings/privacy_page/security_page.html
+@@ -64,202 +64,12 @@
+         pointer-events: auto;
+       }
+     </style>
+-    <picture>
+-      <source
```

Comment:
This will remove one safe browsing section, but I think there are other ui pieces that refer to safe browsing

Reply:
here 
![image](https://github.com/user-attachments/assets/8b212f76-b9bd-4945-9b2d-2fca0fd1a613)

Reply:
oh, right. nevermind